.packit.yaml

```diff
@@ -5,15 +5,18 @@ synced_files:
 upstream_package_name: fmf
 downstream_package_name: fmf
 
+# Epel9 fails to build with dynamic version. Need to create archive with PKG-INFO
+# F37 works with setuptools_scm 7.0
 actions:
   create-archive:
```

Comment:
We might need to add `- hatch run docs:man` (or however we'll generate man) before build, same as in tmt.  
Would be cool to drop man completely, but it didn't fly in tmt, so assuming it won't fly here either.

Reply:
It is included in the `spec` file
```
cp docs/header.txt man.rst
tail -n+7 README.rst >> man.rst
rst2man man.rst > fmf.1
```

For me I think `sdist` should be as vanilla as close as possible to the git archive. But regarding the man generation, we should see about if we want to change it in https://github.com/teemtee/tmt/pull/3088.

We still have quite a few discrepancies as `tmt` has evolved, so how about handling these one step at a time.

Reply:
It's tricky, because then you have discrepancies between rpm source and pypi source. It's a "hack" either way. No strong opinion from me.

Reply:
Thanks.